server/aws-lsp-codewhisperer/src/language-server/agenticChat/agenticChatController.test.ts

```diff
@@ -1081,7 +1081,7 @@ describe('AgenticChatController', () => {
             const typedChatResult = chatResult as ResponseError<ChatResult>
             assert.strictEqual(
                 typedChatResult.data?.body,
-                'Too much context loaded. Please start a new conversation or ask about specific files.'
+                'Too much context loaded. I have cleared the conversation history. Please retry your request.'
```

Comment:
should we mention something along the lines of "with smaller input"?

Reply:
Added. We may need to modify this message entirely after Ankit's input